bitwarden-desktop: 2025.7.0 -> 2025.8.1

r-ryantm opened the pull request:
Automatic update generated by [nixpkgs-update](https://github.com/ryantm/nixpkgs-update) tools. This update was made based on information from passthru.updateScript.


meta.description for bitwarden-desktop is: Secure and free password manager for all of your devices

meta.homepage for bitwarden-desktop is: https://bitwarden.com

meta.changelog for bitwarden-desktop is: https://github.com/bitwarden/clients/releases/tag/desktop-v2025.8.0


###### Updates performed
- Ran passthru.UpdateScript

###### To inspect upstream changes

- [Release on GitHub](https://github.com/bitwarden/clients/releases/tag/desktop-v2025.8.0)

- [Compare changes on GitHub](https://github.com/bitwarden/clients/compare/desktop-v2025.7.0...desktop-v2025.8.0)

###### Impact

<b>Checks done</b>

---

- built on NixOS
- The tests defined in `passthru.tests`, if any, passed
- found 2025.8.0 with grep in /nix/store/7dswld1zx3p1a9r7b4ahbpviqd1girwz-bitwarden-desktop-2025.8.0
- found 2025.8.0 in filename of file in /nix/store/7dswld1zx3p1a9r7b4ahbpviqd1girwz-bitwarden-desktop-2025.8.0

---

<details>
<summary>
<b>Rebuild report</b> (if merged into master) (click to expand)
</summary>

```
3 total rebuild path(s)

3 package rebuild(s)

First fifty rebuilds by attrpath

bitwarden
bitwarden-desktop
```

</details>

<details>
<summary>
<b>Instructions to test this update</b> (click to expand)
</summary>

---

Either **download from the cache**:
```
nix-store -r /nix/store/7dswld1zx3p1a9r7b4ahbpviqd1girwz-bitwarden-desktop-2025.8.0 \
  --option binary-caches 'https://cache.nixos.org/ https://nixpkgs-update-cache.nix-community.org/' \
  --option trusted-public-keys '
  nixpkgs-update-cache.nix-community.org-1:U8d6wiQecHUPJFSqHN9GSSmNkmdiFW7GW7WNAnHW0SM=
  cache.nixos.org-1:6NCHdD59X431o0gWypbMrAURkbJ16ZPMQFGspcDShjY=
  '
```
(The nixpkgs-update cache is only trusted for this store-path realization.)
For the cached download to work, your user must be in the `trusted-users` list or you can use `sudo` since root is effectively trusted.

Or, **build yourself**:
```
nix-build -A bitwarden-desktop https://github.com/r-ryantm/nixpkgs/archive/934055cb0fdcad8316e051fe66957693820d3be2.tar.gz
```
Or:
```
nix build github:r-ryantm/nixpkgs/934055cb0fdcad8316e051fe66957693820d3be2#bitwarden-desktop
```

After you've downloaded or built it, look at the files and if there are any, run the binaries:
```
ls -la /nix/store/7dswld1zx3p1a9r7b4ahbpviqd1girwz-bitwarden-desktop-2025.8.0
ls -la /nix/store/7dswld1zx3p1a9r7b4ahbpviqd1girwz-bitwarden-desktop-2025.8.0/bin
```

---

</details>
<br/>



### Pre-merge build results

We have automatically built all packages that will get rebuilt due to
this change.

This gives evidence on whether the upgrade will break dependent packages.
Note sometimes packages show up as _failed to build_ independent of the
change, simply because they are already broken on the target branch.

## `nixpkgs-review` result

Generated using [`nixpkgs-review`](https://github.com/Mic92/nixpkgs-review).

Command: `nixpkgs-review`

---
### `x86_64-linux`
<details>
  <summary>:white_check_mark: 1 package built:</summary>
  <ul>
    <li>bitwarden-desktop</li>
  </ul>
</details>

---

###### Maintainer pings

cc @amarshall for [testing](https://github.com/ryantm/nixpkgs-update/blob/main/doc/nixpkgs-maintainer-faq.md#r-ryantm-opened-a-pr-for-my-package-what-do-i-do).

> [!TIP]
> As a maintainer, if your package is located under `pkgs/by-name/*`, you can comment **`@NixOS/nixpkgs-merge-bot merge`** to automatically merge this update using the [`nixpkgs-merge-bot`](https://github.com/NixOS/nixpkgs-merge-bot).

---

Add a :+1: [reaction] to [pull requests you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[pull requests you find important]: https://github.com/NixOS/nixpkgs/pulls?q=is%3Aopen+sort%3Areactions-%2B1-desc

r-ryantm committed:
bitwarden-desktop: 2025.7.0 -> 2025.8.0

iedame commented:
## `nixpkgs-review` result

Generated using [`nixpkgs-review-gha`](https://github.com/Defelo/nixpkgs-review-gha)

Command: `nixpkgs-review pr 436873`
Commit: [`934055cb0fdcad8316e051fe66957693820d3be2`](https://github.com/NixOS/nixpkgs/commit/934055cb0fdcad8316e051fe66957693820d3be2) ([subsequent changes](https://github.com/NixOS/nixpkgs/compare/934055cb0fdcad8316e051fe66957693820d3be2..pull/436873/head))
Merge: [`35fa1ce8b554d557ea70ac3fb6edb1d7ce20fb28`](https://github.com/NixOS/nixpkgs/commit/35fa1ce8b554d557ea70ac3fb6edb1d7ce20fb28)

Logs: https://github.com/iedame/nixpkgs-review-gha/actions/runs/17237170873


---
### `x86_64-linux`
<details>
  <summary>:white_check_mark: 1 package built:</summary>
  <ul>
    <li>bitwarden-desktop</li>
  </ul>
</details>


---
### `aarch64-linux`
<details>
  <summary>:white_check_mark: 1 package built:</summary>
  <ul>
    <li>bitwarden-desktop</li>
  </ul>
</details>


---
### `x86_64-darwin` (sandbox = true)
<details>
  <summary>:white_check_mark: 1 package built:</summary>
  <ul>
    <li>bitwarden-desktop</li>
  </ul>
</details>


---
### `aarch64-darwin` (sandbox = true)
<details>
  <summary>:white_check_mark: 1 package built:</summary>
  <ul>
    <li>bitwarden-desktop</li>
  </ul>
</details>

iedame reviewed (approved):
LGTM, builds okay.

SuperSandro2000 commented:
> maybe we could bump to 2025.8.1 even

Yep, please.

SuperSandro2000 committed:
bitwarden-desktop: 2025.8.0 -> 2025.8.1

Diff: https://github.com/bitwarden/clients/compare/desktop-v2025.8.0...desktop-v2025.8.1

Changelog: https://github.com/bitwarden/clients/releases/tag/desktop-v2025.8.1
